src/pepr/uds-operator-config/values.yaml

```diff
@@ -10,6 +10,7 @@ operator:
   UDS_LOG_LEVEL: "###ZARF_VAR_UDS_LOG_LEVEL###"
   AUTHSERVICE_REDIS_URI: "###ZARF_VAR_AUTHSERVICE_REDIS_URI###"
   KUBEAPI_CIDR: ""
+  KUBENODE_CIDRS: ""
```

Comment:
Let's make sure we document this with an example in [this doc](https://github.com/defenseunicorns/uds-core/blob/970-all-nodes/docs/reference/configuration/uds-networking-configuration.md), probably a `KubeNodes` section.

Reply:
I added a section on `KubeNodes` in 1d4f0af8